### DISCUSSION TITLE
Internet Connection

### DISCUSSION THREAD

**Discussion by MateusDornelles** [ltsp/ltsp#562] [Q&A]
Hi, can I have internet connection on a LTSP server with just one NIC ? 

Thanks,
Mateus.

**Comment by alkisg** [comment]
Of course, that's the default setup.

**Comment by MateusDornelles** [comment]
Sorry, my bad, I forgot to mention a lot of details, I have a server with a single NIC and it's running the DHCP. I configured it with ltsp dnsmasq --proxy-dhcp=0 and assigned the static ip 192.168.67.1 to my connection. It works fine, but i would like to be able to access the internet from the server, using a single nic and without an external dhcp, i dont know if it's possible.

**Comment by alkisg** [comment]
What Internet device are you using, a router?

What is its IP? It needs to be the gateway of the LTSP server.
